python/src/opendp/_convert.py

```diff
@@ -364,3 +367,41 @@ def _slice_to_hashmap(raw: FfiSlicePtr) -> Dict[Any, Any]:
 
 def _wrap_in_slice(ptr, len_: int) -> FfiSlicePtr:
     return FfiSlicePtr(FfiSlice(ctypes.cast(ptr, ctypes.c_void_p), len_))
+
+
+#                            (output         , input       )
+CallbackFn = ctypes.CFUNCTYPE(ctypes.c_void_p, AnyObjectPtr)
```

Comment:
Shouldn't this return `FFIResult` (rather than a pointer) like other FFI functions?

Reply:
That would be nice, but we're running into a limitation of Python here:
https://bugs.python.org/issue5710#msg85731

I've added a comment to this effect.

Reply:
Dang it!